arch/risc-v/include/esp32c3/irq.h

```diff
@@ -229,4 +229,8 @@
 
 #define NR_IRQS  (RISCV_NIRQ_INTERRUPTS + ESP32C3_NIRQ_PERIPH + ESP32C3_NIRQ_GPIO)
 
+/* Section for exception handler. */
+
+#define EXCEPTION_SECTION .iram1
```

Comment:
should we put to chip.h like setintstack?

Reply:
Done.